README.md

```diff
@@ -1,22 +1,28 @@
-# ember-async-component
+ember-async-component
+==============================================================================
 
 This ember addon is used for building promise aware container components which
 handles success, loading and error states. This component also works seemlessly
 with [ember-cli-fastboot](https://github.com/ember-fastboot/ember-cli-fastboot)
 
-## Compatibility
+Compatibility
+------------------------------------------------------------------------------
 
-- Ember.js v3.16 or above
-- Ember CLI v3.12 or above
-- Node.js v10 or above
+* Ember.js v3.16 or above
+* Ember CLI v2.13 or above
```

Comment:
@elwayman02, I just noticed this change and I don't think we want to support any version before 3.12... I think that it would be like "use it at your own risk" if it works.
Our ember-try isn't testing this old version either, so I don't think we should claim this.

Reply:
@alias-mac this is Ember CLI, not Ember. We don't ever test multiple versions of Ember CLI. FWIW, the standard Ember blueprint provides this change, so it's easier to just take that rather than try to manually change it unless there is a very specific reason we're doing something that certain versions of Ember CLI won't support. Even all of our internal addons are documented this way, technically. :)

Reply:
Yes, I was highlighting the fact that we don't have a matrix test in CI to run with `2.13` of ember-cli, so technically there isn't anything that validates that this compatibility is true. Since the documentation mentioned v3.12 or above (for CLI), I believe we should keep it to avoid confusion (or add a test in CI to validate that ember cli 2.13 works too).

Reply:
There isn't anything that validates 3.12 is compatible either.